deepchem/feat/molecule_featurizers/one_hot_featurizer.py

```diff
@@ -17,13 +19,13 @@
 
 
 class OneHotFeaturizer(MolecularFeaturizer):
```

Comment:
Outside the scope of this PR, but we probably need a `StringFeaturizer` abstract class at some point. Ideally we should have an inheritance structure like

```
class OneHotFeaturizer(StringFeaturizer, MolecularFeaturizer):
```

Reply:
I just made a PR for this :) https://github.com/deepchem/deepchem/pull/2464